homeassistant/components/alexa/capabilities.py

```diff
@@ -124,11 +128,13 @@ def __init__(
         self,
         entity: State,
         instance: str | None = None,
+        config: AbstractConfig | None = None,
         non_controllable_properties: bool | None = None,
     ) -> None:
         """Initialize an Alexa capability."""
         self.entity = entity
         self.instance = instance
+        self.config = config
```

Comment:
Why is this needed?

Reply:
This is needed because i read the locale from the configuration.
Since we should stick to the english locale in this PR (as you said in the next comment) this will be removed.

Reply:
I reverted the commit which adds this line.